tests/hstestcase.py

```diff
@@ -63,13 +66,24 @@ def _delete_job(cls, jobkey):
 
     @classmethod
     def _set_testbotgroup(cls):
-        cls.project.settings.apipost(jl={'botgroups': [cls.testbotgroup]})
+        for botgroup in cls.testbotgroups:
+            # Settings
+            # endpoint wouldn't accept list > 1
+            cls.project.settings.apipost(jl={'botgroups': [botgroup]})
```

Comment:
i mean running `cls.project.settings.apipost(jl={'botgroups': [botgroup]})` for each botgroup is unnecessary, as each subsequent operation overrides the output of the previous one.